main.go

```diff
@@ -170,6 +170,25 @@ func formatSeverity() string {
 	return w.String()
 }
 
+func validateLinters(linters map[string]*Linter, config *Config) error {
+	var unknownLinters []string
+	for name, _ := range linters {
+		if _, isDefault := defaultLinters[name]; !isDefault {
+			if _, isCustom := config.Linters[name]; !isCustom {
+				unknownLinters = append(unknownLinters, name)
+			}
+		}
+	}
```

Comment:
I guess this will fix the current problem, but wont handle the case where someone defines a linter and forgets to specify a `Command`.  Maybe instead of checking isDefault/isCustom this could call `validateLinter()` which could check that any required fields are set?

Reply:
Yeah, I wanted to keep the scope of this one super-focused.  I'm only detecting when you give gometalinter a linter that it doesn't know about.

This change also doesn't address the case where you define a linter with all the correct flags, but an invalid command.  I was thinking this is probably a follow on change.